io/src/main/scala/fs2/io/file/pulls.scala

```diff
@@ -22,6 +23,22 @@ object pulls {
       h: FileHandle[F]): Pull[F, Byte, Unit] =
     _readRangeFromFileHandle0(chunkSize, start, end)(h)
 
+  def keepReadingFromFileHandle[F[_]: Timer](chunkSize: Int, delay: FiniteDuration)(
+      h: FileHandle[F]): Pull[F, Byte, Unit] =
+    _keepReadingFromFileHandle0(chunkSize, 0, delay)(h)
+
+  private def _keepReadingFromFileHandle0[F[_]](
+      chunkSize: Int,
+      offset: Long,
+      delay: FiniteDuration)(h: FileHandle[F])(implicit timer: Timer[F]): Pull[F, Byte, Unit] =
+    Pull.eval(h.read(chunkSize, offset)).flatMap {
+      case Some(bytes) =>
+        Pull.output(bytes) >> _keepReadingFromFileHandle0(chunkSize, offset + bytes.size, delay)(h)
+      case None =>
+        timer.sleep(delay)
```

Comment:
Note this `timer.sleep(delay)` isn't being used -- change to `Pull.eval_(timer.sleep(delay)) >> _keepReadingFromFileHandle0(...)`

Reply:
Thanks! Done.